### DISCUSSION TITLE
Better documentation

### DISCUSSION THREAD

**Discussion by kasir-barati** [golevelup/nestjs#845] [Ideas]
Hey @underfisk I just wanted to know if I should open a PR regarding some working examples for different exchanges?

Related issues: 
1. https://github.com/golevelup/nestjs/issues/780
2. https://github.com/golevelup/nestjs/issues/786

Recently I was looking for a working example for Topic exchange and different scenarios that we can have so I ended up [here](https://github.com/golevelup/nestjs/issues/315#issuecomment-2396140928). But I guess it would be much much easier for other devs to see some sort of docs in the official repo, do not you agree?

So I think we can create an example directory in the root of this repo, something like this: `./examples/rabbitmq/topic`.

Any comment or thought on this subject?

**Comment by underfisk** [comment]
Documentation improvements are always welcomed and appreciated @kasir-barati 
Feel free to make your proposal and we'll review it. I think you make a good point for people that's getting into RMQ and its better if they can surface a collection of example(s)/guides

**Comment by underfisk** [comment]
We have launched a new documentation [page](https://golevelup.github.io/nestjs/) feel free to give feedback!

**Reply by kasir-barati** [reply]
Will do, I was so occupied for the past few months I did not find any chance to open a PR :smiling_face_with_tear:. But thanks for the improvements, I will definitely take a look.
